.github/workflows/publish_packages.yaml

```
@@ -0,0 +1,27 @@
+name: publish_packages
+
+on:
+  push:
+    branches: [main]
+    tags:
+    - "[a-zA-Z0-9_-]+-v[0-9]+.[0-9]+.[0-9]+"
```

actionlint (1.7.4)

invalid glob pattern. unexpected character ']' while checking character range in []. end of range is missing. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)